### PR TITLE
 milestone: Need for submission

### PR DESCRIPTION
**Pull request by hemangsk**
Improves List item alignment
screen shot 2017-01-20 at 9 43 39 pm

Adds initiatives and suborg in overview
screen shot 2017-01-20 at 10 08 43 pm

Admins (Description message needs polishing)
screen shot 2017-01-20 at 11 39 10 pm

FAQ Markdown 
screen shot 2017-01-20 at 11 45 22 pm

### CODE REVIEW

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/109/commits/aeffef6790f7169c9feaf6aadf6111f51f58b15e

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/109/commits/539439699aec64c5dc6c95acf715b647c36b9d1b

File: data/faq.js [hunk 1]
```diff
@@ -52,25 +52,28 @@ The earlier you start, the better!"
 },
 {
 	"ques": "What are my obligations as a student during the GSoC?",
-	"ans" : "We expect you to:\
-\
-- Be active on [gitter](https://coala.io/chat).\
```
Comment (sils) [inline data/faq.js]: rather try using `*`s? It works with the milestones md right?

**Comment by sils** [comment]
unack https://github.com/coala/projects/pull/109/commits/404f3680a3b7f174f158035cadb9dec8d4143d58

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/109/commits/b8e6b33e21e70ca46f42ed0fd19e8528224cc9f5

File: data/admins.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+admins = {
```
Comment (sils) [inline data/admins.js]: other order please, sims, jayvdb, sils, fneu. Also won't a simple list do? Still has ordering.

**Comment by sils** [comment]
unack https://github.com/coala/projects/pull/109/commits/07331c1c09dac5ea52e89085cf54ccc915f6d606

**Comment by hemangsk** [comment]
reack 9a3f25e 02aef08 ce816b2

File: index.html [hunk 3]
```diff
@@ -4,13 +4,14 @@
 <head>
 
     <title>coala</title>
-
+
```
Comment (sils) [inline index.html]: why those spaces?

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/109/commits/ab0b0bd9c757b6c7fa2d4cb9c3f0825eaba8c81a

**Comment by hemangsk** [comment]
reack 96fe7f1 56e8954 735b837

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/109/commits/387314f2063a4611db03f023507348cc9f9f1d4a

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/109/commits/acedc83e7287b65c4b865b49e73b67884c72b10c

**Comment by sils** [comment]
@rultor merge

**Pull request merged**